webhooks/ssp_webhook_test.go

```diff
@@ -202,7 +202,7 @@ var _ = Describe("SSP Validation", func() {
 			table.Entry("no namepsace provided", "", "test-name", false),
 			table.Entry("no name provided", internal.GoldenImagesNamespace, "", true),
 			table.Entry("golden image namespace provided", internal.GoldenImagesNamespace, "test-name", false),
-			table.Entry("invalid namespace provided", "invalid-namespace", "test-name", true),
+			table.Entry("other namespace provided", "other-namespace", "test-name", false),
```

Comment:
The webhook no longer checks namespace, so you can remove all code that tests it.